README.md

```diff
@@ -72,66 +95,126 @@ For command line help, visit our wiki page on Bourbon’s [command line interfac
   mv app/assets/stylesheets/application.css app/assets/stylesheets/application.scss
   ```
 
-1. Delete _all_ Sprockets directives in `application.scss` (`require`, `require_tree` and `require_self`) and use Sass’s native `@import` instead ([why?](http://pivotallabs.com/structure-your-sass-files-with-import)).
+1. Delete _all_ Sprockets directives in `application.scss` (`require`,
+   `require_tree` and `require_self`) and use Sass’s native `@import` instead
+   ([why?][sass-import]).
 
-1. Import Bourbon at the beginning of `application.scss`. All additional stylesheets should be imported below Bourbon:
+1. Import Bourbon at the beginning of `application.scss`. All additional
+   stylesheets should be imported below Bourbon:
 
   ```scss
   @import "bourbon";
   @import "home";
   @import "users";
   ```
 
-  [Help! I’m getting an undefined mixin error.](https://github.com/thoughtbot/bourbon/wiki/Rails-Help-%5C-Undefined-mixin)
+  [sass-import]: http://pivotallabs.com/structure-your-sass-files-with-import
 
-## Installing with npm and using a Node-based asset pipeline
+### Installing with npm and using a Node-based asset pipeline
 
 1. Add Bourbon as a dependency:
 
   ```bash
   npm install --save bourbon
   ```
 
-1. If you’re using [Eyeglass](http://eyeglass.rocks), skip to Step 3. Otherwise, you’ll need to add Bourbon to your node-sass `includePaths` option. `require("bourbon").includePaths` is an array of directories that you should pass to node-sass. How you do this depends on how node-sass is integrated into your project.
+1. If you’re using [eyeglass], skip to Step 3. Otherwise,
+you’ll need to add Bourbon to your node-sass `includePaths` option.
+`require("bourbon").includePaths` is an array of directories that you should
+pass to node-sass. How you do this depends on how node-sass is integrated into
+your project.
 
 1. Import Bourbon into your Sass files:
 
   ```scss
   @import "bourbon";
   ```
 
-## Installing older versions of Bourbon
+  [eyeglass]: http://eyeglass.rocks
+
+### Installing older versions of Bourbon
 
 1. Uninstall any Bourbon gem versions you already have:
 
   ```bash
   gem uninstall bourbon
   ```
 
-1. Reinstall the Bourbon gem, using the `-v` flag to specify the version you need:
+1. Reinstall the Bourbon gem, using the `-v` flag to specify the version
+   you need:
 
   ```bash
   gem install bourbon -v 3.2.4
   ```
 
-1. Follow the [instructions above](#installation) to install Bourbon into your project.
+1. Follow the [instructions above](#installation) to install Bourbon into
+   your project.
+
+## Command Line Interface
+
+```bash
+bourbon [options]
+```
+
+### Options
+
+Describe the available commands:
```

Comment:
I could see this as a table. Maybe something like:

|command | verbose |output|
|----------|--------|---------|
|-h | --help | Describe the available commands. |
|-v | --version | Show the installed Bourbon version. |